emma/allennlp_classes/ontoemma_model.py

```diff
@@ -58,9 +101,9 @@ def _average_nonzero(t_stack):
 
         return torch.stack(output_rows)
 
-
     @overrides
     def forward(self,  # type: ignore
+                lr_features: Dict[str, torch.LongTensor],
```

Comment:
nit: The name "lr_features" is a bit confusing cuz these can be used for any classifier. Maybe `manual_features` or `sparse_features`?